### PR TITLE
[hi] Localize DevSecOps

### PR DESCRIPTION
**Pull request by SahityaGupta25**
Fixes: #727

### CODE REVIEW

File: content/hi/DevSecOps.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+| Title                                    | Status    | Category | Exclude_Search |
+| ---------------------------------------- | --------- | -------- | -------------- |
+| [DevOps](https://github.com/cncf/glossary/blob/main/content/hi/devops.md) ( डेवॉप्स) | Completed | अवधारणा  | True           |
```
Comment (jihoon-seo) [inline content/hi/DevSecOps.md:3]: Please use https://github.com/cncf/glossary/blob/main/content/hi/_TEMPLATE.md?plain=1 as a markdown template.

```suggestion
---
title: DevOps (डेवॉप्स)
status: Completed
category: अवधारणा
exclude_search: true
---
```

File: content/hi/DevSecOps.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+#यह क्या है
+
+सॉफ्टवेयर विकास जीवनचक्र के हर चरण में [DevOps](https://github.com/cncf/glossary/blob/main/content/hi/devops.md) स्वचालित रूप से सुरक्षा में साथ निभाना है | जिससे Agile और [DevOps](https://github.com/cncf/glossary/blob/main/content/hi/devops.md) की गति से सुरक्षित सॉफ़्टवेयर का विकास संभव हो जाता है।
```
Comment (jihoon-seo) [inline content/hi/DevSecOps.md:7]:
```suggestion
सॉफ्टवेयर विकास जीवनचक्र के हर चरण में [DevOps](/hi/devops/) स्वचालित रूप से सुरक्षा में साथ निभाना है | जिससे Agile और [DevOps](/hi/devops/) की गति से सुरक्षित सॉफ़्टवेयर का विकास संभव हो जाता है।
```

File: content/hi/DevSecOps.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+| Title                                    | Status    | Category | Exclude_Search |
```
Comment (jihoon-seo) [inline content/hi/DevSecOps.md:1]: Could you rename this file to `content/hi/devsecops.md` (with no uppercase)?

File: content/hi/DevSecOps.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+1. [DevOps](https://github.com/cncf/glossary/blob/main/content/hi/devops.md) टीमें सुरक्षा को एक उपद्रव के रूप में देखती हैं
+
+2. IT सुरक्षा दल [DevOps](https://github.com/cncf/glossary/blob/main/content/hi/devops.md) की तेज़ गति के साथ तालमेल नहीं बिठा पा रहे हैं
+
+3. कई ओपन-सोर्स और अपरिपक्व टूल में अपर्याप्त सुरक्षा सुविधाएं होती हैं
+
+4. अपर्याप्त रूप से प्रबंधित विशेषाधिकार प्राप्त अभिगम नियंत्रण के परिणामस्वरूप हमले के अधिक अवसर मिलते हैं
+
+   #समाधान
```
Comment (jihoon-seo) [inline content/hi/DevSecOps.md:22]:
```suggestion
## समाधान
```

File: content/hi/DevSecOps.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
+| Title                                    | Status    | Category | Exclude_Search |
+| ---------------------------------------- | --------- | -------- | -------------- |
+| [DevOps](https://github.com/cncf/glossary/blob/main/content/hi/devops.md) ( डेवॉप्स) | Completed | अवधारणा  | True           |
+
+#यह क्या है
+
+सॉफ्टवेयर विकास जीवनचक्र के हर चरण में [DevOps](https://github.com/cncf/glossary/blob/main/content/hi/devops.md) स्वचालित रूप से सुरक्षा में साथ निभाना है | जिससे Agile और [DevOps](https://github.com/cncf/glossary/blob/main/content/hi/devops.md) की गति से सुरक्षित सॉफ़्टवेयर का विकास संभव हो जाता है।
+
+#समस्या
```
Comment (jihoon-seo) [inline content/hi/DevSecOps.md:9]:
```suggestion
## समस्या
```

File: content/hi/DevSecOps.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
+| Title                                    | Status    | Category | Exclude_Search |
+| ---------------------------------------- | --------- | -------- | -------------- |
+| [DevOps](https://github.com/cncf/glossary/blob/main/content/hi/devops.md) ( डेवॉप्स) | Completed | अवधारणा  | True           |
+
+#यह क्या है
```
Comment (jihoon-seo) [inline content/hi/DevSecOps.md:5]:
```suggestion
## यह क्या है
```

File: content/hi/DevSecOps.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+आज की डिजिटल दुनिया में आईटी सुरक्षा एक महत्वपूर्ण मुद्दा है, और खतरे रातोंरात दूर नहीं होंगे। साइबर हमले और धोखाधड़ी बढ़ रही है। इस कठोर वास्तविकता का सामना करते हुए, यह समझ से बाहर है कि आज कोई भी संगठन DevSecOps कार्यप्रणाली के सुरक्षा पहलू की उपेक्षा करेगा।
+[DevOps](https://github.com/cncf/glossary/blob/main/content/hi/devops.md) सुरक्षा की चुनौतियों को संक्षेप में प्रस्तुत करने के लिए:
+
+1. [DevOps](https://github.com/cncf/glossary/blob/main/content/hi/devops.md) टीमें सुरक्षा को एक उपद्रव के रूप में देखती हैं
+
+2. IT सुरक्षा दल [DevOps](https://github.com/cncf/glossary/blob/main/content/hi/devops.md) की तेज़ गति के साथ तालमेल नहीं बिठा पा रहे हैं
+
+3. कई ओपन-सोर्स और अपरिपक्व टूल में अपर्याप्त सुरक्षा सुविधाएं होती हैं
+
+4. अपर्याप्त रूप से प्रबंधित विशेषाधिकार प्राप्त अभिगम नियंत्रण के परिणामस्वरूप हमले के अधिक अवसर मिलते हैं
```
Comment (jihoon-seo) [inline content/hi/DevSecOps.md:20]: In [English version](https://glossary.cncf.io/devsecops/), there is no numbered list.

Is it right that you localized from the [English version](https://glossary.cncf.io/devsecops/)?

File: content/hi/microservices.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
+| Title                                    | Status    | Category |
+| ---------------------------------------- | --------- | -------- |
+| [Microservices](https://github.com/cncf/glossary/blob/main/content/en/microservices.md) | Completed | Concept  |
```
Comment (jihoon-seo) [inline content/hi/microservices.md:3]:
```suggestion
---
title: Microservices
status: Completed
category: अवधारणा
exclude_search: true
---
```

File: content/hi/microservices.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+
+# यह क्या है
+
+```
```
Comment (jihoon-seo) [inline content/hi/microservices.md:7]: Please remove this line containing 3 backticks.

```suggestion
```

File: content/hi/microservices.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+माइक्रोसर्विसेज अनुप्रयोग विकास के लिए एक आधुनिक दृष्टिकोण है जो क्लाउड नेटिव तकनीकों का लाभ उठाता है। जबकि आधुनिक एप्लिकेशन, जैसे नेटफ्लिक्स, एक एकल ऐप प्रतीत होते हैं, वे वास्तव में छोटी सेवाओं का एक संग्रह हैं - सभी एक साथ मिलकर काम कर रहे हैं। उदाहरण के लिए, एक एकल पृष्ठ जो आपको वीडियो तक पहुंचने, खोजने और पूर्वावलोकन करने की अनुमति देता है, संभवतः छोटी सेवाओं द्वारा संचालित होता है जो प्रत्येक इसके एक पहलू को संभालते हैं (जैसे आपके ब्राउज़र में खोज, प्रमाणीकरण और पूर्वावलोकन चलाना)। संक्षेप में, माइक्रोसर्विसेज एक एप्लिकेशन आर्किटेक्चर पैटर्न को संदर्भित करता है जो आमतौर पर  मोनोलिथिक एप्लिकेशन के विपरीत होता है।
+
+```
```
Comment (jihoon-seo) [inline content/hi/microservices.md:10]: Please remove this line containing 3 backticks.

```suggestion
```

File: content/hi/microservices.md [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+
+#समस्या यह संबोधित करता है
+
+[Micro Services](https://github.com/cncf/glossary/blob/main/content/en/microservices.md) अखंड अनुप्रयोगों द्वारा उत्पन्न चुनौतियों की प्रतिक्रिया है। आम तौर पर, किसी एप्लिकेशन के विभिन्न हिस्सों को अलग-अलग स्केल करने की आवश्यकता होगी। एक ऑनलाइन स्टोर, उदाहरण के लिए, चेकआउट की तुलना में अधिक उत्पाद दृश्य होने वाला है। इसका मतलब है कि आपको चेकआउट की तुलना में चल रहे उत्पाद दृश्य कार्यक्षमता की अधिक प्रतियों की आवश्यकता होगी। एक अखंड अनुप्रयोग में, तर्क के उन बिट्स को अलग से तैनात नहीं किया जा सकता है। यदि आप व्यक्तिगत रूप से उत्पाद की कार्यक्षमता को माप नहीं सकते हैं, तो आपको पूरे ऐप को उन सभी घटकों के साथ डुप्लिकेट करना होगा जिनकी आपको आवश्यकता नहीं है - संसाधनों का एक अक्षम उपयोग। मोनोलिथिक एप्लिकेशन भी डेवलपर्स के लिए डिजाइन के नुकसान के आगे झुकना आसान बनाते हैं। चूंकि सभी कोड एक ही स्थान पर हैं, इसलिए उस कोड को कसकर युग्मित करना आसान है और चिंताओं को अलग करने के सिद्धांत को लागू करना कठिन है। मोनोलिथ को अक्सर डेवलपर्स को उत्पादक होने से पहले पूरे कोडबेस को समझने की आवश्यकता होती है।
+
+# यह कैसे मदद करता है
+
+```
```
Comment (jihoon-seo) [inline content/hi/microservices.md:18]: Please remove this line containing 3 backticks.

```suggestion
```

File: content/hi/microservices.md [hunk 12]
```diff
@@ -0,0 +1,20 @@
...
+विभिन्न माइक्रोसर्विसेज में कार्यक्षमता को अलग करने से उन्हें स्वतंत्र रूप से तैनात, अपडेट और स्केल करना आसान हो जाता है। अलग-अलग टीमों को बड़े एप्लिकेशन के अपने छोटे हिस्से पर ध्यान केंद्रित करने की अनुमति देकर आप उनके लिए बाकी संगठन को नकारात्मक रूप से प्रभावित किए बिना अपने ऐप्स पर काम करना आसान बनाते हैं। जबकि माइक्रोसर्विसेज कई समस्याओं का समाधान करते हैं, वे परिचालन ओवरहेड भी बनाते हैं - वे चीजें जिन्हें आपको तैनात करने और परिमाण या अधिक के क्रम में वृद्धि का ट्रैक रखने की आवश्यकता होती है। कई क्लाउड-देशी तकनीकों का उद्देश्य माइक्रोसर्विसेज को तैनात और प्रबंधित करना आसान बनाना है।
+```
```
Comment (jihoon-seo) [inline content/hi/microservices.md:20]: Please remove this line containing 3 backticks.

```suggestion
```

File: content/hi/microservices.md [hunk 13]
```diff
@@ -0,0 +1,20 @@
+| Title                                    | Status    | Category |
+| ---------------------------------------- | --------- | -------- |
+| [Microservices](https://github.com/cncf/glossary/blob/main/content/en/microservices.md) | Completed | Concept  |
+
+# यह क्या है
```
Comment (jihoon-seo) [inline content/hi/microservices.md:5]:
```suggestion
## यह क्या है
```

File: content/hi/microservices.md [hunk 14]
```diff
@@ -0,0 +1,20 @@
...
+
+```
+माइक्रोसर्विसेज अनुप्रयोग विकास के लिए एक आधुनिक दृष्टिकोण है जो क्लाउड नेटिव तकनीकों का लाभ उठाता है। जबकि आधुनिक एप्लिकेशन, जैसे नेटफ्लिक्स, एक एकल ऐप प्रतीत होते हैं, वे वास्तव में छोटी सेवाओं का एक संग्रह हैं - सभी एक साथ मिलकर काम कर रहे हैं। उदाहरण के लिए, एक एकल पृष्ठ जो आपको वीडियो तक पहुंचने, खोजने और पूर्वावलोकन करने की अनुमति देता है, संभवतः छोटी सेवाओं द्वारा संचालित होता है जो प्रत्येक इसके एक पहलू को संभालते हैं (जैसे आपके ब्राउज़र में खोज, प्रमाणीकरण और पूर्वावलोकन चलाना)। संक्षेप में, माइक्रोसर्विसेज एक एप्लिकेशन आर्किटेक्चर पैटर्न को संदर्भित करता है जो आमतौर पर  मोनोलिथिक एप्लिकेशन के विपरीत होता है।
+
+```
+
+#समस्या यह संबोधित करता है
```
Comment (jihoon-seo) [inline content/hi/microservices.md:12]:
```suggestion
## समस्या यह संबोधित करता है
```

File: content/hi/microservices.md [hunk 15]
```diff
@@ -0,0 +1,20 @@
...
+
+[Micro Services](https://github.com/cncf/glossary/blob/main/content/en/microservices.md) अखंड अनुप्रयोगों द्वारा उत्पन्न चुनौतियों की प्रतिक्रिया है। आम तौर पर, किसी एप्लिकेशन के विभिन्न हिस्सों को अलग-अलग स्केल करने की आवश्यकता होगी। एक ऑनलाइन स्टोर, उदाहरण के लिए, चेकआउट की तुलना में अधिक उत्पाद दृश्य होने वाला है। इसका मतलब है कि आपको चेकआउट की तुलना में चल रहे उत्पाद दृश्य कार्यक्षमता की अधिक प्रतियों की आवश्यकता होगी। एक अखंड अनुप्रयोग में, तर्क के उन बिट्स को अलग से तैनात नहीं किया जा सकता है। यदि आप व्यक्तिगत रूप से उत्पाद की कार्यक्षमता को माप नहीं सकते हैं, तो आपको पूरे ऐप को उन सभी घटकों के साथ डुप्लिकेट करना होगा जिनकी आपको आवश्यकता नहीं है - संसाधनों का एक अक्षम उपयोग। मोनोलिथिक एप्लिकेशन भी डेवलपर्स के लिए डिजाइन के नुकसान के आगे झुकना आसान बनाते हैं। चूंकि सभी कोड एक ही स्थान पर हैं, इसलिए उस कोड को कसकर युग्मित करना आसान है और चिंताओं को अलग करने के सिद्धांत को लागू करना कठिन है। मोनोलिथ को अक्सर डेवलपर्स को उत्पादक होने से पहले पूरे कोडबेस को समझने की आवश्यकता होती है।
+
+# यह कैसे मदद करता है
```
Comment (jihoon-seo) [inline content/hi/microservices.md:16]:
```suggestion
## यह कैसे मदद करता है
```

File: content/hi/microservices.md [hunk 16]
```diff
@@ -0,0 +1,20 @@
...
+
+```
+
+#समस्या यह संबोधित करता है
+
+[Micro Services](https://github.com/cncf/glossary/blob/main/content/en/microservices.md) अखंड अनुप्रयोगों द्वारा उत्पन्न चुनौतियों की प्रतिक्रिया है। आम तौर पर, किसी एप्लिकेशन के विभिन्न हिस्सों को अलग-अलग स्केल करने की आवश्यकता होगी। एक ऑनलाइन स्टोर, उदाहरण के लिए, चेकआउट की तुलना में अधिक उत्पाद दृश्य होने वाला है। इसका मतलब है कि आपको चेकआउट की तुलना में चल रहे उत्पाद दृश्य कार्यक्षमता की अधिक प्रतियों की आवश्यकता होगी। एक अखंड अनुप्रयोग में, तर्क के उन बिट्स को अलग से तैनात नहीं किया जा सकता है। यदि आप व्यक्तिगत रूप से उत्पाद की कार्यक्षमता को माप नहीं सकते हैं, तो आपको पूरे ऐप को उन सभी घटकों के साथ डुप्लिकेट करना होगा जिनकी आपको आवश्यकता नहीं है - संसाधनों का एक अक्षम उपयोग। मोनोलिथिक एप्लिकेशन भी डेवलपर्स के लिए डिजाइन के नुकसान के आगे झुकना आसान बनाते हैं। चूंकि सभी कोड एक ही स्थान पर हैं, इसलिए उस कोड को कसकर युग्मित करना आसान है और चिंताओं को अलग करने के सिद्धांत को लागू करना कठिन है। मोनोलिथ को अक्सर डेवलपर्स को उत्पादक होने से पहले पूरे कोडबेस को समझने की आवश्यकता होती है।
```
Comment (jihoon-seo) [inline content/hi/microservices.md:14]: - "Microservices" is a single-word term.
- It seems that the link to the term itself is not necessary.

```suggestion
Microservices अखंड अनुप्रयोगों द्वारा उत्पन्न चुनौतियों की प्रतिक्रिया है। आम तौर पर, किसी एप्लिकेशन के विभिन्न हिस्सों को अलग-अलग स्केल करने की आवश्यकता होगी। एक ऑनलाइन स्टोर, उदाहरण के लिए, चेकआउट की तुलना में अधिक उत्पाद दृश्य होने वाला है। इसका मतलब है कि आपको चेकआउट की तुलना में चल रहे उत्पाद दृश्य कार्यक्षमता की अधिक प्रतियों की आवश्यकता होगी। एक अखंड अनुप्रयोग में, तर्क के उन बिट्स को अलग से तैनात नहीं किया जा सकता है। यदि आप व्यक्तिगत रूप से उत्पाद की कार्यक्षमता को माप नहीं सकते हैं, तो आपको पूरे ऐप को उन सभी घटकों के साथ डुप्लिकेट करना होगा जिनकी आपको आवश्यकता नहीं है - संसाधनों का एक अक्षम उपयोग। मोनोलिथिक एप्लिकेशन भी डेवलपर्स के लिए डिजाइन के नुकसान के आगे झुकना आसान बनाते हैं। चूंकि सभी कोड एक ही स्थान पर हैं, इसलिए उस कोड को कसकर युग्मित करना आसान है और चिंताओं को अलग करने के सिद्धांत को लागू करना कठिन है। मोनोलिथ को अक्सर डेवलपर्स को उत्पादक होने से पहले पूरे कोडबेस को समझने की आवश्यकता होती है।
```

**Review by jihoon-seo — COMMENTED** [review]
Previews:
- https://deploy-preview-760--cncfglossary.netlify.app/hi/devsecops/
- https://deploy-preview-760--cncfglossary.netlify.app/hi/microservices/

@SahityaGupta25 PTAL!

**Comment by seokho-son** [comment]
Hi @SahityaGupta25 Hindi localization team is using `dev-hi` branch for localization contribution.

@anubha-v-ardhan could you guide @SahityaGupta25 to change the base and target branches to `dev-hi` ? :)
(note that, the contributor needs to change the base branch in Github GUI as well as change base branch in contributor's feature branch)

**Comment by Shivansh-yadav13** [comment]
- This PR will also close #728

**Comment by anubha-v-ardhan** [comment]
> * This PR will also close [[hi] Translate `Microservices` into `Hindi` #728](https://github.com/cncf/glossary/issues/728)

We can accept this PR but please try to do one page per PR in future. It helps your PR to get reviewed and merged faster and us to easily manage the localization effort :)

File: content/hi/DevSecOps.md (hunk 4 above)
Comment (jayesh-srivastava) [inline content/hi/DevSecOps.md:22]: Here `समाधान` is wrong. It should be `यह कैसे मदद करता है`

File: content/hi/DevSecOps.md [hunk 17]
```diff
@@ -0,0 +1,34 @@
...
+
+   #समाधान
+
+DevSecOps  के लाभों को ध्यान में रखते हुए, इसे अभी भी व्यापक रूप से लागू नहीं किया जा रहा है। कम से कम, अभी तक। आइए DevSecOps  को अपनाने के लाभों के बारे में गहराई से जानें:
```
Comment (jayesh-srivastava) [inline content/hi/DevSecOps.md:24]: @SahityaGupta25 This whole section doesn't seem like the correct localisation from the English version. Please use https://github.com/cncf/glossary/blob/main/content/en/devsecops.md as reference.

**Review by jayesh-srivastava — CHANGES_REQUESTED** [review]

**Comment by jayesh-srivastava** [comment]
@SahityaGupta25 Please consider using separate PRs for separate pages. It will be easier to review and merge.
PTAL @anubha-v-ardhan

**Comment by SahityaGupta25** [comment]
@Garima-Negi Sure but I don't know how to remove can you guide me how to remove pages ??

**Comment by Garima-Negi** [comment]
Closing this PR it is obsolete and had 2 different terms being localized leading to confusion.
I have synced with @SahityaGupta25  he will be working on microservices in a separate PR

**Pull request closed without merging**